### DISCUSSION TITLE
[remark-cli] Remark-lint CLI - can you specify a folder?

### DISCUSSION THREAD

**Discussion by emdashdrupal** [remarkjs/remark#796] [Q&A] [answered]
We're testing remark-lint from the CLI, and finding that it doesn't check files outside its own folder structure. For example, running `npm run lint "h:\projects\docs"`  and various iterations always returns no errors. If I copy those files to the remark folder, they're all linted.

I've also installed remark-cli and get same results by typing remark "h:\projects\docs". 

Is this expected behavior?

Thanks!

**Comment by wooorm** [comment, marked as answer]
Hi there!

You should be able to do that, yes.
But now you’re passing it to `npm run`, not the `remark`.
You should be able to specify a folder in your actual `lint` script.

And I might be wrong but I think `npm run` lets you pass arguments to the underlying script by using two dashes (this is a unix thing though): `npm run lint -- "h:\projects\docs"`

**Reply by emdashdrupal** [reply]
this is great. i'll give it a try. thank you! we have it working in vs code now and it's already making a difference.
